platform/mds/mds-web/pom.xml

```diff
@@ -53,6 +53,11 @@
             <groupId>commons-fileupload</groupId>
             <artifactId>commons-fileupload</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.google.code.gson</groupId>
+            <artifactId>gson</artifactId>
+            <version>2.6.1</version>
```

Comment:
in modules and motech repo we are using com.google.code.gson from artifact 'org.motechproject.com.google.code.gson'. The version is always defined in main pom 'motech.pom or modules.pom', in child we are only defining groupId and artifactId. You can check the following poms to look how to correctly add dependency into this module :
 openmrs/pom.xml, message-campaign/pom.xml, schedule-tracking/pom.xml